examples/physics/run_coala.py

```
@@ -67,10 +67,7 @@ def run_kernel_case(kernel):
     elif kernel == 1:
         dthydro = 1e-2
         ndthydro = 300
-    elif kernel == 2:
-        dthydro = 1e-1
-        ndthydro = 500
-    elif kernel == 3:
+    elif kernel == 2 or kernel == 3:
```

Ruff (0.16.0)

[warning] Consider merging multiple comparisons: `kernel in {2, 3}`.

Merge multiple comparisons

(PLR1714)